CMakeLists.txt

```diff
@@ -57,13 +54,15 @@ set(BOOSTPY ${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
 find_package(Boost COMPONENTS date_time filesystem python${BOOSTPY} numpy${BOOSTPY} program_options system unit_test_framework REQUIRED)
 include_directories(${Boost_INCLUDE_DIR})
 
-find_path(LOFAR_STATION_RESPONSE_DIR NAMES StationResponse/Station.h)
-find_library(LOFAR_STATION_RESPONSE_LIB stationresponse)
-if(LOFAR_STATION_RESPONSE_DIR AND LOFAR_STATION_RESPONSE_LIB)
-  include_directories(${LOFAR_STATION_RESPONSE_DIR})
-  add_definitions(-DHAVE_LOFAR_BEAM)
-  message(STATUS "LOFAR beam library found.")
-
+find_path(EVERYBEAM_INCLUDE_DIR NAMES EveryBeam/load.h)
+if(NOT EVERYBEAM_INCLUDE_DIR)
+  message(FATAL_ERROR "EveryBeam include file (EveryBeam/load.h) not found: please install https://git.astron.nl/RD/EveryBeam")
+endif()
+find_library(EVERYBEAM_LIB everybeam)
+if(EVERYBEAM_INCLUDE_DIR AND EVERYBEAM_LIB)
+  include_directories(${EVERYBEAM_INCLUDE_DIR})
+  message(STATUS "EveryBeam library found.")
+# TODO everybeam is no longer optional, so these files can become unconditional
   set(LOFAR_DEPENDENT_FILES
```

Comment:
Is this list complet @aroffringa ? Found a couple more EveryBeam dependencies, i.e.:

```
#       modified:   DPPP/ApplyBeam.cc
#       modified:   DPPP/DemixWorker.cc
#       modified:   DPPP/MSReader.cc
#       modified:   DPPP/Predict.cc
```

Reply:
Do you mean the LOFAR_DEPENDENT_FILES list? DPPP/MSReader.cc was not dependent on the presence of the beam, so I don't understand your question. These files can move to the regular target files.